packages/system/src/Location/ListView/Toolbar.tsx

```diff
@@ -63,6 +64,12 @@ export const Toolbar: FC<{
     }
   };
 
+  const showDeleteConfirmation = useConfirmationModal({
+    onConfirm: deleteAction,
+    message: t('messages.confirm-delete-locations'),
+    title: t('heading.are-you-sure'),
+  });
+
```

Comment:
Had to break the pattern for this one -- the deleteAction determines if it can delete by just trying the mutation and catching the error, so can't verify if `canDelete` before running the actual delete. 🤷‍♂️